pkg/servicerole/servicerole.go

```diff
@@ -61,9 +63,19 @@ func createServiceRole(namespace, role string, policy *zms.Policy) (model.Config
 		Namespace: namespace,
 	}
 
-	methods := make([]string, 0)
+	pathToMethods := make(map[string][]string)
 	for _, assertion := range policy.Assertions {
-		methods = append(methods, strings.ToUpper(assertion.Action))
+		key := emptyPath
+		path := strings.Split(assertion.Resource, ":")
```

Comment:
comment with example how its defined on athen policy section

Reply:
Added the comment.